libcxx/docs/TestingLibcxx.rst

```diff
@@ -330,13 +330,48 @@ additional headers.
           taken to make it work in earlier language versions.
 
 
-Additional reading
-------------------
-
-The function ``CxxStandardLibraryTest`` in the file
-``libcxx/utils/libcxx/test/format.py`` has documentation about writing test. It
-explains the difference between the test named  ``foo.pass.cpp`` and named
-``foo.verify.cpp`` are.
+Test names
```

Comment:
I would like to avoid duplicating this documentation. It will get out of date when we add new test formats in the future.

I would suggest either migrating the whole documentation for our test format into `.rst` or dropping this change.

Reply:
Okay!! Honestly, you tell me!! Which one would you prefer? I am happy to migrate it all and do a cleanup if you think it would be helpful!

Reply:
I think documenting the libc++ testing format in `.rst` would be more helpful to contributors than it staying hidden where it's currently at. I've had to show it to people several times in the past so I think this is a common one that people don't know about. I would still link to these docs from `format.py` though just to connect them in an obvious way.